ts/Dashboards/PluginHandler.ts

```diff
@@ -39,21 +38,35 @@ namespace PluginHandler {
      * */
 
     export interface DashboardPlugin<T = (AnyRecord|undefined)> {
+        /** @internal */
         custom: T;
+        /**
+         * Maximal version of plugin that is compatible with dashboard
```

Comment:
Do we even use it anywhere?

Reply:
@bre1470 ?

Reply:
I do not understand the question either. It is not used by a plugin, because there is no discontinued plugin yet. The property is used though.